app/assets/stylesheets/recipe_new.scss

```diff
@@ -0,0 +1,101 @@
+@import "colors";
+
+.recipe-name-input-block textarea {
+  font-size: 16pt;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, color, font-family, font-size